reference/supportability.md

```diff
@@ -0,0 +1,159 @@
+## Supportability
+
+The supportability tool collects Mayastor specific information from the cluster using the [kubectl plugin](https://mayastor.gitbook.io/introduction/v/develop/reference/kubectl-plugin) command-line tool. It uses the dump command, which interacts with the Mayastor services to build an archive (ZIP) file that acts as a placeholder for the bundled information.
+
+
+### Using the supportability tool
+
+To bundle Mayastor's complete system information, execute:
+
+{% tab title="Command" %}
+```text
+kubectl mayastor dump system -n mayastor -d <output_directory_path>
+```
+{% endtab %}
+
+{% hint style="info" %}
+**Note:** If the nodes exist on a private network or the node ports are not exposed to an outside cluster, all the above commands need to pass these additional parameters:
+kubectl mayastor dump <resource> -d <output_directory> -r http://127.0.0.1:30011 -l http://127.0.0.1:3100 -e http://127.0.0.1:2379 -n mayastor
```

Comment:
How will this work since there are loopback IP addresses?

Reply:
I think this example is taken from previous way of usage where we used to `kubectl port-forward` these ports and use them. That's why its all loopback. Now we don't need to specify any of these and it should work as we do internal port-forward.